app/components/common/ConfirmModal.js

```diff
@@ -73,7 +73,7 @@ ConfirmModal.propTypes = {
   footer: PropTypes.string,
   confirm: PropTypes.func.isRequired,
   close: PropTypes.func,
-  modal: PropTypes.object.isRequired
+  modal: PropTypes.object
```

Comment:
Just double checking: is this property not required anymore? It seems like the only purpose of this component is to display this modal.

Reply:
This was throwing a warning after some updates for the team. Not sure. But you can review it later. Thanks